lib/spack/spack/build_systems/intel.py

```diff
@@ -123,55 +860,79 @@ def configure(self, spec, prefix):
+# ARCH_SELECTED                                {IA32, INTEL64, ALL}
+#
+# * Mentioned but unexplained in check.awk:
+#
+# NO_VALIDATE   (?!)
+#
+# * Only for licensed incarnations (obviously):
+# ACTIVATION_SERIAL_NUMBER                     {, snpat}
+# ACTIVATION_LICENSE_FILE                      {, lspat, filepat}
+# ACTIVATION_TYPE                              {exist_lic, license_server,
+#                                              license_file, trial_lic,
+#                                              serial_number}
+#
+# * Only for Amplifier (obviously):
+# AMPLIFIER_SAMPLING_DRIVER_INSTALL_TYPE       {build, kit}
+# AMPLIFIER_DRIVER_ACCESS_GROUP                {, anythingpat, vtune}
+# AMPLIFIER_DRIVER_PERMISSIONS                 {, anythingpat, 666}
+# AMPLIFIER_LOAD_DRIVER                        {yes, no}
+# AMPLIFIER_C_COMPILER                         {, filepat, auto, none}
+# AMPLIFIER_KERNEL_SRC_DIR                     {, filepat, auto, none}
+# AMPLIFIER_MAKE_COMMAND                       {, filepat, auto, none}
+# AMPLIFIER_INSTALL_BOOT_SCRIPT                {yes, no}
+# AMPLIFIER_DRIVER_PER_USER_MODE               {yes, no}
+#
+# * Only for MKL and Studio:
+# CLUSTER_INSTALL_REMOTE                       {yes, no}
+# CLUSTER_INSTALL_TEMP                         {, filepat}
+# CLUSTER_INSTALL_MACHINES_FILE                {, filepat}
+#
+# * -- Not -- for MKL, and possibly others:
+# PHONEHOME_SEND_USAGE_DATA                    {yes, no}
+#
+# * "backward compatibility" (?)
+# INSTALL_MODE                                 {RPM, NONRPM}
+# download_only                                {yes}
+# download_dir                                 {, filepat}
+
         config = {
-            # Accept EULA, valid values are: {accept, decline}
             'ACCEPT_EULA': 'accept',
-
-            # Optional error behavior, valid values are: {yes, no}
             'CONTINUE_WITH_OPTIONAL_ERROR': 'yes',
-
-            # Install location, valid values are: {/opt/intel, filepat}
             'PSET_INSTALL_DIR': prefix,
-
-            # Continue with overwrite of existing installation directory,
-            # valid values are: {yes, no}
             'CONTINUE_WITH_INSTALLDIR_OVERWRITE': 'yes',
-
-            # List of components to install,
-            # valid values are: {ALL, DEFAULTS, anythingpat}
             'COMPONENTS': ';'.join(self._filtered_components),
-
-            # Installation mode, valid values are: {install, repair, uninstall}
             'PSET_MODE': 'install',
-
-            # Directory for non-RPM database, valid values are: {filepat}
             'NONRPM_DB_DIR': prefix,
-
-            # Perform validation of digital signatures of RPM files,
-            # valid values are: {yes, no}
             'SIGNING_ENABLED': 'no',
-
-            # Select target architecture of your applications,
-            # valid values are: {IA32, INTEL64, ALL}
             'ARCH_SELECTED': 'ALL',
         }
 
+        if 'mkl' not in self.name:
```

Comment:
It looks like Intel's MPI does not like this either, can you please add that exception too?

Reply:
Addressed – ``PHONEHOME_SEND_USAGE_DATA`` is actually subject to ``if self.license_required``, just like it was before I factored out the ``_determine_license_type()`` tokens.